### ISSUE TITLE
Broken aria-describledby reference on component descriptions

### ISSUE THREAD

**Issue by bjosttveit** [Altinn/app-frontend-react#2441]
### Description of the bug

It looks like we are missing the id on the description element of the component labels. Components have an `aria-describedby=description-nodeId` but the id is not set in the `LabelContent` component.

<img width="437" alt="image" src="https://github.com/user-attachments/assets/6f71682b-eb16-4d9c-b8f7-8e4572d7d2e9">

### Steps To Reproduce

Use wave to check accessibility

**Comment by adamhaeger** [comment]
@paal2707 has already solved this as part of the new datepicker, assigning to him, this issue should be tested when that branch is merged.

**Comment by bjosttveit** [comment]
@adamhaeger when was this fixed? I still have the issue in the latest release

![Image](https://github.com/user-attachments/assets/fe184bbd-3d45-499f-ba53-5f3af53c075f)

**Comment by bjosttveit** [comment]
@paal2707 Has this been fixed since v4.11? Or is this still present in main?

**Comment by paal2707** [comment]
> [@paal2707](https://github.com/paal2707) Has this been fixed since v4.11? Or is this still present in main?

I had a solution that i did not include, because @cammiida already had merged a solution to this.

**Comment by olemartinorg** [comment]
Latest app-frontend looks fine now:
![Image](https://github.com/user-attachments/assets/cc9db7f7-08a7-48c8-bee4-6f704228b1a1)
